zookeeper-api/src/main/java/org/apache/helix/zookeeper/zkclient/ZkClient.java

```diff
@@ -636,7 +636,7 @@ public String create(final String path, Object datat, final List<ACL> acl, final
    * @throws IllegalArgumentException if called from anything else except the ZooKeeper event thread
    * @throws ZkException if any zookeeper exception occurs
    */
-  private String create(final String path, final Object dataObject, final List<ACL> acl,
+   private String create(final String path, final Object dataObject, final List<ACL> acl,
```

Comment:
Nit, Do you want it to be `public` or `private`? If private, no need to add the space.

Reply:
Fixed